src/cli-sdk/src/commands/config.ts

```diff
@@ -191,12 +192,47 @@ const set = async (conf: LoadedConfig) => {
   let parsed: ConfigFileData | null = null
   try {
     parsed = conf.jack.parseRaw(pairs.map(kv => `--${kv}`)).values
-  } catch {
-    throw error('Invalid config keys', {
-      code: 'ECONFIG',
-      found: pairs.map(kv => kv.split('=')[0]),
-      wanted: getSortedKeys(),
-    })
+  } catch (er) {
+    const { cause } = er as ErrorWithCauseObject
```

Comment:
I think it's better to use the `isErrorWithCause` type guard from `error-cause` here and have it just `throw er` at the end if its not. The only reason is to protect against head-scratching dev time errors where the `try` block throws for some other reason.

Reply:
Right, but then we still don't know that `cause` is a cause _object_, just that it's not undefined.

I can still add the guard, but the `as ErrorWithCauseObject` will have to stay, unless we add a `isErrorWithCauseObject` to `@vltpkg/error` (which isn't a terrible idea, of course).